From peterhuewe@gmx.de  Fri Dec 18 10:26:23 2009
From: Peter Huewe <peterhuewe@gmx.de>
Date: Tue, 15 Dec 2009 06:37:36 +0100
Subject: Staging: panel: Adjust range for PANEL_KEYPAD in Kconfig
To: Greg Kroah-Hartman <greg@kroah.com>
Cc: devel@linuxdriverproject.org, Willy Tarreau <w@1wt.eu>, Sudhakar Rajashekhara <sudhakar.raj@ti.com>, Costantino Leandro <lcostantino@gmail.com>, linux-kernel@vger.kernel.org, Peter Huewe <peterhuewe@gmx.de>
Message-ID: <1260855456-25012-1-git-send-email-peterhuewe@gmx.de>


In panel.c there are only the values 0-3 defined. So 4 is invalid:

Signed-off-by: Peter Huewe <peterhuewe@gmx.de>
Cc: Willy Tarreau <w@1wt.eu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/panel/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/panel/Kconfig
+++ b/drivers/staging/panel/Kconfig
@@ -47,7 +47,7 @@ config PANEL_PROFILE
 config PANEL_KEYPAD
 	depends on PANEL && PANEL_PROFILE="0"
 	int "Keypad type (0=none, 1=old 6 keys, 2=new 6 keys, 3=Nexcom 4 keys)"
-	range 0 4
+	range 0 3
 	default 0
 	---help---
 	  This enables and configures a keypad connected to the parallel port.
